resources/Materials/TestSuite/pbrlib/multioutput/multioutput.mtlx

```diff
@@ -76,4 +76,35 @@
     <output name="out" type="color3" nodename="multiply1" />
   </nodegraph>
 
+  <!-- Test custom node implementation with multioutput in a material context -->
+  <color4split name="color4split1" type="multioutput">
+    <input name="in" type="color4" value="1.0, 0.25, 0.25, 0.42" />
+  </color4split>
+  <output name="color4split1_colour" type="color3" nodename="color4split1" output="colour"/>
+  <output name="color4split1_alpha" type="float" nodename="color4split1" output="alpha"/>
+  <material name="multioutput_test5">
+    <shaderref name="standard_surface1" node="standard_surface">
+      <bindinput name="base_color" type="color3" output="color4split1_colour" />
```

Comment:
Can we update the pruning for the findRenderableElements() utility which scans for renderable elements so that it also excludes outputs which are not part of an output graph but are referened by bindinputs. I think when outputs got promoted to be able to be top level we didn't update this logic. Without the prune we'll file the shadereref output + color4split1_colour + color4split1_alpha as renderable outputs.

The outputs in the node graph case is already handled. 

Thanks.

Reply:
Yes that would be good, as we are currently validating/rendering too many elements. More testing is good of course but it also takes more time to run the suite. Will look into fixing this.

Reply:
Yeah. I actually want to prune this some more to not retest embedded nodedefs that are used in shaderrefs but haven't gotten around to it. Thanks.